packages/components/app/styles/components/form/character-count.scss

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// FORM > CHARACTER-COUNT
+//
+
+.hds-form-character-count {
+  display: block;
```

Comment:
do we need this? 🤔
```suggestion
```

Reply:
not required anymore (if we're going forward with a `<div>` elements, as it currently is) – removed in a7ed41d